### PR TITLE
Existing single Cluster Observability Pattern with AWS Mixed approach services

### PR DESCRIPTION
**Pull request by freschri**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Existing EKS Cluster AWS Mixed Observability Accelerator
+
+## Objective
+
+This pattern depends on the previous deployment of [_Single New EKS Cluster Observability Accelerator_](./single-new-eks-cluster.md).
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md:5]: Objective is not spelled out correctly. Objective is to create observability with a mixture of AWS native and Open source managed AWS services on an existing EKS cluster. Also talk about what this mixed observability provides to single new mixed doc.

File: docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+After deploying this pattern you will have the same setup as in [_Single New EKS Cluster AWS Mixed Observability Accelerator_](./single-new-eks-mixed-observability.md).
+
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md:7]: pre req i mentioned i smissing
1. EKS Cluster pointing to single eks cluster as example.
2. OIDC provider setup for EKS cluster. (Single EKS Cluster takes care of it).

File: docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Deploying
+
+1. Follow the instructions in [_Single New EKS Cluster Observability Accelerator_](./single-new-eks-cluster.md).
+
+2. Edit `~/.cdk.json` by setting the kubectl role name, as provided by the output of the above deployment of the _Single New EKS Cluster Observability Accelerator_.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md:12]: Why there is an _ before and after the name in end.

File: docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+```json
+...
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md:15]: you should also callout clusterName should be updated with your EKS Cluster Name

File: docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Existing EKS Cluster AWS Mixed Observability Accelerator
+
+## Objective
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md:3]: Talk about the pattern and diagram will help. You can reuse mixed diagram

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@freschri Looks good overall. Have some feedback on doc

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+Ensure that you have installed the following tools on your machine:
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [cdk](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+4. [npm](https://docs.npmjs.com/cli/v8/commands/npm-install)
+
+You will also need:
+
+1. either an existing EKS cluster, or you can setup a new one with  [Single New EKS Cluster Observability Accelerator](./single-new-eks-cluster.md)
```
Comment (elamaran11) [inline docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md:30]: either -> Either.

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+2. an OpenID Connect (OIDC) provider, associated to the above EKS cluster (note: Single EKS Cluster Pattern takes care of that for you)
```
Comment (elamaran11) [inline docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md:31]: an -> An. note -> Note

**Review by elamaran11 — COMMENTED** [review]
@freschri LGTM, almost there. Couple of minor comments. Once you have it. I will run and will merge. Also fix the GH warning lint issues.

**Review by elamaran11 — APPROVED** [review]
LGTM. I will run this once as e2e and will merge. Thankyou for the PR

**Comment by elamaran11** [comment]
@freschri Please fix the lint warnings, i see broken links.

**Comment by freschri** [comment]
> @freschri Please fix the lint warnings, i see broken links.

done

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md (hunk 7 above)
Comment (freschri) [reply docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md]: done

File: docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md (hunk 6 above)
Comment (freschri) [reply docs/patterns/existing-eks-observability-accelerators/existing-eks-mixed-observability.md]: done

File: docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md (hunk 5 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md]: done

**Review by elamaran11 — APPROVED** [review]
LGTM. I was able to successfully validate this pattern. Nice work @freschri

**Pull request merged**